HealthcareSystem/Backend/Repository/ExaminationRepository/MySqlExaminationRepository/MySqlExaminationRepository.cs

```diff
@@ -148,7 +148,16 @@ public ICollection<Examination> GetExaminationsByPatientAndDateTime(int patientC
                 throw new DatabaseException("The database connection is down.");
             }
         }
-
+        public void DeleteExaminationRepository(int id)
+        {
+            Examination examination = _context.Examinations.SingleOrDefault(d => d.Id == id);
+            if (examination != null) 
+            {
+                _context.Remove(examination);
+                _context.SaveChanges();
+            }
```

Comment:
Why delete the Examination? They can already be cancelled.

Reply:
We have changed it.

Reply:
Use the Cancel method that already exists in the ExaminationService class, or preferably the cancel endpoint in the ScheduleService. There is no need to reimplement this functionality on this branch.

Reply:
Okay, l have changed it.